### DISCUSSION TITLE
where to get version 1.9.1?

### DISCUSSION THREAD

**Discussion by chchia** [theophanemayaud/video-simili-duplicate-cleaner#124] [General]
![image](https://github.com/theophanemayaud/video-simili-duplicate-cleaner/assets/1291771/c2332097-be7a-4a13-a085-dca6f8ab74cc)

Hi i am paid user, but my Microsoft Store is showing i installed 1.9.0 and no update option. can you tell me where to get 1.9.1? and possibly beta test version?

**Comment by theophanemayaud** [comment]
Hi, I’ll have to check what’s going on. Thanks for the info. I’ll get back to you asap.

**Comment by chchia** [comment]
hi any update on this?

**Comment by theophanemayaud** [comment]
Hey @chchia did you receive the 1.10 version on windows ? Sorry for the delay but it should have gone out a few weeks ago !
